qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}-app/pom.xml

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>{{ cookiecutter.project_slug }}</artifactId>
+        <groupId>life.qbic</groupId>
+        <version>{{ cookiecutter.version }}</version> <!-- <<QUBE_FORCE_BUMP>> -->
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>war</packaging>
+    <artifactId>{{ cookiecutter.project_slug }}-app</artifactId>
+    <dependencies>
```

Comment:
![Bildschirmfoto 2021-06-11 um 16 31 10](https://user-images.githubusercontent.com/29627977/121702858-88ae4f80-cad2-11eb-84bc-44d3d6636efb.png)
I just noticed that the url is referring to the app module and not the parent which breaks the linkage: 
One could think about adding the parent module url here akin to the commit to [sample-tracking](https://github.com/qbicsoftware/sample-tracking-status-overview/pull/3/commits/f5053401c1271ffb4ff03b860a58587036e0a379):

Reply:
I commited your suggestion :)